### DISCUSSION TITLE
Select a max stride of 128 in the GUI

### DISCUSSION THREAD

**Discussion by olinesn** [talmolab/sleap#1940] [Ideas]
Hello,

With the increase in high-resolution cameras, it's becoming more necessary to increase the number of pixels that the max stride operates across. In the GUI, the largest option is 64, but it's probably time to make 128 an option.

Thanks!
Stefan

**Comment by roomrys** [comment]
We've merged a PR into develop for this feature (thanks to @MweinbergUmass):
- #1941

**Comment by roomrys** [comment]
Long awaited, but finally integrated. This issue has been fixed in the SLEAP 1.4.1 release!
